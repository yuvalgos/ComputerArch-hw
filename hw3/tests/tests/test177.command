./dflow_calc tests/test177.dat tests/test177.in p185 p3 p64 d105 d20 d131 p143 p124 d3 d53 d18 d115 p128 p11 p68 p115 d121 p148 p5 p99 d172 d10 p163 d10 p150 p61 d189 d26 d75 p150 p154 d76 p47 p127 p126 d185 p86 d121 p159 d90 d176 p119 d31 p152 d161 p31 d121 p57 p40 d67 p163 p174 d5 d190 d132 p62 p74 d124 p160 p190 p177 d20 d7 p136 d137 d22 d13 d46 p33 d38 d5 p198 p199 d143 d94 p115 d11 p92 p188 d60 d147 d106 p159 p134 p190 d131 p22 p98 p100 d101 d30 d107 d54 p102 d146 p194 p101 d183 p63 p108 d76 d132 d178 d54 d28 d153 d174 p31 p44