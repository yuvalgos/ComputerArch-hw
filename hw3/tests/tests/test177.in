8 7 14 9
4 22 25 14
1 31 14 25
6 8 13 10
8 14 2 27
7 0 30 18
4 24 14 28
3 20 27 22
1 25 3 13
5 19 1 6
2 22 20 28
2 11 14 5
7 3 23 14
5 30 12 30
6 15 7 7
0 20 10 18
8 13 12 5
7 19 11 3
8 13 0 20
3 14 27 23
7 25 9 29
0 12 22 17
7 21 16 8
3 27 13 16
3 19 1 25
4 22 13 20
3 6 0 31
1 15 11 24
7 13 18 3
0 3 7 12
6 21 9 2
6 31 6 30
2 29 26 9
1 5 26 16
5 20 0 18
3 10 19 26
6 15 5 0
7 19 11 6
7 26 12 7
4 6 25 14
4 31 9 24
8 16 22 24
7 10 12 14
6 12 3 3
2 14 11 10
5 20 22 30
5 14 11 4
0 24 6 6
0 24 9 23
1 13 31 1
0 11 30 1
2 7 14 4
3 17 5 16
3 6 6 1
1 11 20 7
1 28 10 14
6 6 12 7
4 8 28 1
7 7 5 7
1 8 25 22
0 19 2 4
6 1 17 17
4 23 25 15
1 9 1 6
8 27 17 1
7 10 16 7
8 0 27 22
5 20 21 17
0 20 27 18
3 10 25 29
5 11 26 31
3 1 30 0
1 15 5 6
5 24 5 28
0 23 29 21
6 9 27 2
7 15 19 4
1 28 0 17
8 7 18 0
7 20 15 11
8 27 29 13
8 11 1 29
4 11 21 25
8 15 9 15
4 5 27 7
0 8 4 21
0 11 17 18
1 30 22 2
2 3 31 21
1 7 28 12
6 9 27 30
2 30 27 0
6 19 4 26
6 10 8 5
2 25 31 10
8 0 14 29
0 26 6 8
0 28 16 13
2 16 25 11
0 21 24 12
0 3 9 1
6 16 7 17
8 27 31 0
0 12 26 30
2 7 31 23
1 3 22 28
4 14 25 19
7 15 0 3
0 3 10 13
6 28 25 17
3 31 25 8
6 6 1 21
6 14 28 13
8 30 12 8
8 22 12 27
2 15 6 2
2 8 25 4
5 29 15 14
1 13 31 8
7 15 3 13
1 12 12 26
0 14 1 16
6 15 26 4
6 24 20 3
0 26 20 6
7 17 10 10
3 25 7 20
7 18 12 4
1 22 16 30
1 4 10 1
5 10 19 19
0 12 22 30
4 17 18 13
6 12 2 11
1 3 20 16
6 3 28 8
6 14 7 25
6 6 18 20
8 8 9 10
5 19 15 12
2 30 9 27
4 28 12 20
7 0 18 2
4 12 0 19
8 27 26 12
6 28 22 22
3 29 16 24
4 15 9 16
4 10 5 27
7 17 1 10
4 28 24 11
2 29 30 24
0 2 9 6
6 12 26 7
0 18 3 23
0 31 27 20
2 6 12 16
1 16 13 14
8 26 28 26
5 22 14 3
5 22 21 14
1 26 26 15
6 31 6 12
2 24 29 13
7 15 9 30
5 8 8 14
7 15 27 22
5 2 24 18
5 22 1 24
4 31 20 28
7 9 2 13
8 5 21 1
7 20 5 4
0 13 28 27
3 2 1 16
5 1 22 13
5 4 3 13
1 8 30 27
3 15 27 26
3 8 6 17
7 15 26 19
8 22 17 28
5 21 10 6
2 1 18 26
6 9 25 31
8 5 15 4
7 9 11 15
5 5 24 10
1 3 26 14
1 14 28 18
0 11 31 19
7 8 0 7
1 9 25 9
4 3 13 0
3 22 12 2
8 2 15 18
1 31 30 21
2 19 7 17
7 25 28 7
2 30 25 1
