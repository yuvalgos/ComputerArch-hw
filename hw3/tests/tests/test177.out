Reading the opcodes latency info from tests/test177.dat ... Got latency for 9 opcodes
Reading the program file tests/test177.in ... Found 200 instructions
getProgDepth()==295
getDepDepth(185)==272
getDepDepth(3)==0
getDepDepth(64)==100
getInstDeps(105)=={25,97}
getInstDeps(20)=={-1,-1}
getInstDeps(131)=={128,113}
getDepDepth(143)==194
getDepDepth(124)==170
getInstDeps(3)=={-1,-1}
getInstDeps(53)=={39,-1}
getInstDeps(18)=={5,15}
getInstDeps(115)=={111,-1}
getDepDepth(128)==176
getDepDepth(11)==23
getDepDepth(68)==123
getDepDepth(115)==170
getInstDeps(121)=={71,101}
getDepDepth(148)==228
getDepDepth(5)==0
getDepDepth(99)==60
getInstDeps(172)=={171,129}
getInstDeps(10)=={7,-1}
getDepDepth(163)==221
getInstDeps(10)=={7,-1}
getDepDepth(150)==216
getDepDepth(61)==83
getInstDeps(189)=={150,154}
getInstDeps(26)=={5,2}
getInstDeps(75)=={64,-1}
getDepDepth(150)==216
getDepDepth(154)==236
getInstDeps(76)=={60,-1}
getDepDepth(47)==30
getDepDepth(127)==156
getDepDepth(126)==77
getInstDeps(185)=={180,176}
getDepDepth(86)==83
getInstDeps(121)=={71,101}
getDepDepth(159)==236
getInstDeps(90)=={80,87}
getInstDeps(176)=={135,173}
getDepDepth(119)==212
getInstDeps(31)=={26,13}
getDepDepth(152)==192
getInstDeps(161)=={158,147}
getDepDepth(31)==26
getInstDeps(121)=={71,101}
getDepDepth(57)==47
getDepDepth(40)==34
getInstDeps(67)=={30,52}
getDepDepth(163)==221
getDepDepth(174)==215
getInstDeps(5)=={-1,-1}
getInstDeps(190)=={169,139}
getInstDeps(132)=={127,118}
getDepDepth(62)==36
getDepDepth(74)==17
getInstDeps(124)=={79,111}
getDepDepth(160)==101
getDepDepth(190)==238
getDepDepth(177)==228
getInstDeps(20)=={-1,-1}
getInstDeps(7)=={-1,1}
getDepDepth(136)==84
getInstDeps(137)=={127,79}
getInstDeps(22)=={-1,3}
getInstDeps(13)=={-1,-1}
getInstDeps(46)=={11,-1}
getDepDepth(33)==0
getInstDeps(38)=={21,0}
getInstDeps(5)=={-1,-1}
getDepDepth(198)==227
getDepDepth(199)==258
getInstDeps(143)=={142,139}
getInstDeps(94)=={40,93}
getDepDepth(115)==170
getInstDeps(11)=={4,-1}
getDepDepth(92)==44
getDepDepth(188)==253
getInstDeps(60)=={-1,-1}
getInstDeps(147)=={90,101}
getInstDeps(106)=={94,92}
getDepDepth(159)==236
getDepDepth(134)==169
getDepDepth(190)==238
getInstDeps(131)=={128,113}
getDepDepth(22)==17
getDepDepth(98)==202
getDepDepth(100)==153
getInstDeps(101)=={89,52}
getInstDeps(30)=={-1,-1}
getInstDeps(107)=={95,105}
getInstDeps(54)=={45,51}
getDepDepth(102)==64
getInstDeps(146)=={101,123}
getDepDepth(194)==222
getDepDepth(101)==152
getInstDeps(183)=={154,161}
getDepDepth(63)==100
getDepDepth(108)==197
getInstDeps(76)=={60,-1}
getInstDeps(132)=={127,118}
getInstDeps(178)=={144,161}
getInstDeps(54)=={45,51}
getInstDeps(28)=={-1,12}
getInstDeps(153)=={124,104}
getInstDeps(174)=={71,157}
getDepDepth(31)==26
getDepDepth(44)==44
